packages/react/src/components/pickers/BasePicker.tsx

```diff
@@ -889,19 +892,32 @@ export class BasePicker<T, P extends IBasePickerProps<T>>
       const selectedSuggestion =
         currentIndex > -1 ? this.suggestionStore.getSuggestionAtIndex(this.suggestionStore.currentIndex) : undefined;
       const selectedSuggestionAlertText = selectedSuggestion ? selectedSuggestion.ariaLabel : undefined;
+      // keeping the id/className here for legacy support
       return (
-        <div
-          className={suggestionAlertClassName}
-          role="alert"
-          id={this._ariaMap.selectedSuggestionAlert}
-          aria-live="assertive"
-        >
-          {selectedSuggestionAlertText}{' '}
+        <div id={this._ariaMap.selectedSuggestionAlert} className={suggestionAlertClassName}>
+          {`${selectedSuggestionAlertText} `}
         </div>
       );
     }
   }
 
+  protected renderCustomAlert(alertClassName: string = legacyStyles.screenReaderOnly) {
```

Comment:
Actually I take back what I said earlier (sorry), I did some searching internally and couldn't find any overrides of the old method. So I think it's okay to just remove it.

Reply:
Aaaaand naturally as soon as I say that, I look at the search results from one more unlikely location and find some usage 🤦‍♀️